doc/api/cli.md

```diff
@@ -52,6 +52,13 @@ If this flag is passed, the behavior can still be set to not abort through
 [`process.setUncaughtExceptionCaptureCallback()`][] (and through usage of the
 `domain` module that uses it).
 
+### `--completion`
+<!-- YAML
+added: REPLACEME
+-->
+
+Output source-able bash completion script for node.
```

Comment:
A more complete example of how to use this would be good here.

Reply:
I'll add an example. Thanks

Reply:
Maybe reword this to:

`Print source-able bash completion script for Node.js.`?

Reply:
Sounds good, I'll update. Sorry for the late reply by the way, I just saw this comment.